.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,22 @@
+name: End to End integration tests
+
+on:
+  - pull_request
+  - push
+  - workflow_dispatch
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Start containers
```

Comment:
I think we need to build the rucio dev container from source before starting the cluster or otherwise tests may fail when a new API endpoint is used or when rucio dependencies change.

Reply:
@bziemons I've modified the workflow as follows:
- build from src + use any cached image layers
- uploading image/cached layers to GitHub registry
- run the tests.

It works in my fork:
https://github.com/maany/rucio/runs/1492936906?check_suite_focus=true 

The access to rucio's git registry seems to be restricted for PRs. 
https://github.com/rucio/rucio/pull/4165/checks?check_run_id=1492937164#step:6:1739 

It's an ongoing issue for GitHub Actions:
https://github.com/actions/first-interaction/issues/10 

It should be fixed when the PR is merged IMO.